src/observers/link_click_observer.ts

```diff
@@ -33,8 +33,13 @@ export class LinkClickObserver {
   }
 
   clickBubbled = (event: MouseEvent) => {
+    // TODO: remove before any merge
+    console.log("event: ", event)
+    console.log("target is: ", event.target)
+    console.log("fixed target is: ", event.composedPath()[0])
     if (this.clickEventIsSignificant(event)) {
-      const link = this.findLinkFromClickTarget(event.target)
+      const target = event.composedPath()[0]
```

Comment:
```suggestion
      const target = (event.composedPath && event.composedPath()[0]) || event.target
```
Having a fallback for unsupported browsers would be nice.